src/current/v24.1/molt-fetch.md

```diff
@@ -275,23 +309,116 @@ To drop existing tables and create new tables before loading the data, use `'dro
 --table-handling 'drop-on-target-and-recreate'
 ~~~
 
-With each option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not exist.
+When using the `'drop-on-target-and-recreate'` option, MOLT Fetch creates a new CockroachDB table to load the source data if one does not already exist. To guide the automatic schema creation, you can [explicitly map source types to CockroachDB types](#type-mapping).
+
+#### Type mapping
+
+If [`'drop-on-target-and-recreate'`](#target-table-handling) is set, MOLT Fetch automatically creates a CockroachDB schema that is compatible with the source data. The column types are determined as follows:
```

Comment:
Sometimes we say "MOLT Fetch" like here, other times we say `molt fetch`. It's a nit, but it might be good to be consistent.

Reply:
I made a few adjustments to hopefully clarify that `molt fetch` pertains directly to the running process, and MOLT Fetch is describing the functionality more generally.